src/services/hasura/hasura.service.ts

```
@@ -38,27 +38,316 @@ export class HasuraService {
+          return !Number.isNaN(userNumGt) && !Number.isNaN(conditionNumGt) && userNumGt > conditionNumGt;
+        }
+
+        default: {
+          // Default to contains/includes behavior
+          const valueIncludes = cleanConditionValues[0].includes(cleanUserValue);
+          const userIncludes = cleanUserValue.includes(cleanConditionValues[0]);
+          return valueIncludes || userIncludes;
+        }
+      }
+    } catch (error) {
+      console.error('Error evaluating eligibility condition:', error.message);
```

ESLint

[error] Replace `······console.error('Error·evaluating·eligibility·condition:',·error.message` with `↹↹↹console.error(⏎↹↹↹↹'Error·evaluating·eligibility·condition:',⏎↹↹↹↹error.message,⏎↹↹↹`

(prettier/prettier)